lib/mina/kubernetes.rb

```diff
@@ -150,3 +157,21 @@ def apply_kubernetes_resources(options)
     command "#{render_cmd} | #{deploy_cmd}"
   end
 end
+
+def apply_global_kubernetes_resources(options)
+  run :local do
+    comment "Applying all global Kubernetes resources..."
+
+    proxy_env = "HTTPS_PROXY=#{fetch(:proxy)}" if fetch(:proxy)
+    filepaths = options&.[](:filepaths) || "config/deploy/#{fetch(:stage)}"
+
+    render_cmd = "#{proxy_env} krane render --bindings=image_repo=#{fetch(:image_repo)},image_tag=#{fetch(:image_tag)},nzamespace=#{fetch(:namespace)} --current_sha #{fetch(:image_tag)} -f #{filepaths}"
+    deploy_cmd = "#{proxy_env} krane global-deploy #{fetch(:kubernetes_context)} --selector app=broadcast-capturer  --stdin "
```

Comment:
If we need to include the app parameter it should be configurable: `app=broadcast-capturer`

Reply:
ohh, missed that one! thanks for pointing out.

Reply:
Is there any other param apart from `app` that we could use as a `selector`?

If we're okay with `app`, I'm thinking to do it like this - 

```
deploy_cmd = "#{proxy_env} krane global-deploy #{fetch(:kubernetes_context)} --selector app=#{fetch(:application_name)}  --stdin "
```
which is set in the `config/deploy.rb` for broadcast-caturer. Not sure what would be ideal for other repos.

Reply:
Do we even need to deploy with the selector?

Reply:
If we do, let's make the senetire selector value configurable.

Reply:
Yes, it's required to deploy with selector.